.github/workflows/pull-request-management.yml

```
@@ -126,12 +126,38 @@ jobs:
         run: |
           pip install -r .github/requirements-ci-dev.txt -r ansible_collections/arista/avd/requirements-dev.txt --upgrade
 
+  # ----------------------------------- #
+  # Build Collection
+  # ----------------------------------- #
+  build_pyavd:
+    name: Build pyavd
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: |
+            3.12
+      - name: Create wheel packages of pyavd dependencies
+        run: |
+          pip install build
+          cd python-avd
+          python -m build --wheel --outdir /tmp/pyavd
+      - name: Compress wheel packages
+        run: tar -czf /tmp/pyavd.tar.gz -C /tmp pyavd
```

Why is not only the wheel file? why do you need the directory?

```
tree pyavd
pyavd
└── pyavd-5.3.0-py3-none-any.whl
```

```suggestion
        run: tar -czf /tmp/pyavd.tar.gz -C /tmp pyavd
```

---

also a wheel is technically a ZIP file so we are taring an archive inside a directory - I think you can just pass the wheel around

---

yes, removed this step and uploading the wheel file directly